### PR TITLE
replace tw with unocss

### PR DESCRIPTION
**Pull request by onmax**

### CODE REVIEW

**Comment by Albermonte** [comment]
The X in the modal's close button is not centered
image

For consistency, I will fill the lightning with a white bg
image

The Nimiq icon is not centered
image

Zooming to more than 14z causes all markers to disappear.

- [Zoom 14](https://deploy-preview-8--nimiq-map.netlify.app/@45.513871339171104,-73.57503056526187,14z)
image

- [Zoom 15](https://deploy-preview-8--nimiq-map.netlify.app/@45.5138713391711,-73.5723698139191,15z)
image


The cuttlery icon seems too big/not centered
image

Hovering ATM markers causes bg to disappear
image

File: apps/web/src/components/BasicInfoLocation.vue [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      </div>
+    </template>
+    <span v-else-if="location.isAtm" row-start-2 font-semibold text="12 neutral-800 inverted:neutral-300">
+      <template v-if="location.accepts?.length > 0 && location.sells?.length > 0">{{
+        $t('Buy & sell crypto')
+      }}</template>
+      <template v-else-if="location.accepts.length > 0">{{ $t('Sell crypto only') }}</template>
+      <template v-else-if="location.sells.length > 0">{{ $t('Buy crypto only') }}</template>
+    </span>
+    <p v-if="location.address" leading-18 :grid="`~ cols-1 row-start-3 ${location.photo ? 'col-span-2' : 'col-span-3'}`"
```
Comment (Albermonte) [inline apps/web/src/components/BasicInfoLocation.vue:32]: Why is this a grid if the content is only text?

Comment (onmax) [reply apps/web/src/components/BasicInfoLocation.vue:32]: You are right. Not needed. probably copy 🍝

**Comment by onmax** [comment]
@Albermonte fyi, the 15z it is caused by the new providers. Oopsie, ~~I will fix it~~ Fixed it

> Error fetching database: 400 . Response {}. Data {"code":"42804","details":"Returned type provider_type does not match expected type text in column 8.","hint":null,"message":"structure of query does not match function result type"}

**Comment by onmax** [comment]
> The Nimiq icon is not centered

 Fun fact. It is not the Nimiq Icon, it is the Nimiq Pay Icon. I need to set the background color 🤓

**Comment by onmax** [comment]
Thanks for you feedback. I will move this to #9

**Pull request closed without merging**